src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-and-dt.mdx

```diff
@@ -46,6 +41,6 @@ Here are some tips for finding and querying data:
   * Query for traces containing at least one back-end app with `appName is not null`.
   * Query for traces containing both mobile and back-end spans by combining the two previous conditions.
 
-## Troubleshooting
+## Troubleshooting [#troubleshooting]
 
 If you don't see end-user spans, or are having other distributed tracing issues, see [Troubleshooting](/docs/apm/distributed-tracing/troubleshooting/troubleshooting-missing-trace-data#browser-data).
```

Comment:
Same comment as in Android draft

Reply:
@barbnewrelic, thanks for checking this over! I think that anchor was a mistake, so I removed it.